.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -26,7 +26,7 @@ Tip: include an error message (in a `<details></details>` tag) if your issue is
 
 ## Reduced test case
 
-The best way to get your bug fixed is to provide a [reduced test case](https://developer.mozilla.org/en-US/docs/Mozilla/QA/Reducing_testcases).
+The best way to get your bug fixed is to provide a [reduced test case](https://web.archive.org/web/20210306034028/https://developer.mozilla.org/en-US/docs/Mozilla/QA/Reducing_testcases).
```

Comment:
The CI pipeline has a dead link linter. I wasn't able to find where the copy moved to on MDN so I linked to the internet archive copy